cli/config.go

```diff
@@ -64,6 +66,8 @@ func registerFlags(c *Config, f *flag.FlagSet) {
 	f.StringVar(&c.CSRFile, "csr", "", "Certificate signature request file for new public key")
 	f.StringVar(&c.CAFile, "ca", "", "CA used to sign the new certificate")
 	f.StringVar(&c.CAKeyFile, "ca-key", "", "CA private key")
+	f.StringVar(&c.EPCAFile, "endpoint-cert", "", "Other endpoint CA used to sign the new certificate")
```

Comment:
I think I would call these `tls-cert` and `tls-key`, with `TLSCertFile` and `TLSKeyFile`